### DISCUSSION TITLE
[Question] Is the web corresponding to the ui docs open source?

### DISCUSSION THREAD

**Discussion by ppxb** [lobehub/lobe-ui#78] [Q&A] [answered]
### 🧐 问题描述 | Proposed Solution

Or the docs web is based on your team's work or others open source framework?
Thanks.

[a comment by an automated account is not shown]

**Comment by canisminor1990** [comment, marked as answer]
MIT License
Check this repo for docs theme: https://github.com/lobehub/dumi-theme-lobehub
